project/urls.py

```diff
@@ -24,10 +24,14 @@
 from exp import urls as exp_urls
 from project import settings
 from web import urls as web_urls
+from osf_oauth2_adapter import views as osf_oauth2_adapter_views
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include(api_urls)),
+    url(r'^accounts/social/login/cancelled/', osf_oauth2_adapter_views.login_errored_cancelled),
+    url(r'^accounts/social/login/error/', osf_oauth2_adapter_views.login_errored_cancelled),
+    url(r'^accounts/', include('allauth.urls')),
```

Comment:
I think there's a class-level variable to set the login page url. I might make a mixin for exp's loginrequiredmixin and set that variable there.